src/Location/Location.tsx

```diff
@@ -1,6 +1,15 @@
-import React from "react";
-import * as ReactDOM from "react-dom";
-import "./Location.scss";
+import React from 'react'
+import './Location.scss'
 
+const Location: React.FC = () => {
```

Comment:
Nice, we can work with the styling on this

Reply:
YES PLEASE! I would love that.